pkg/kthena-router/controller/modelserver_controller_test.go

```diff
@@ -893,6 +893,200 @@ func TestModelServerController_ComprehensiveLifecycleTest(t *testing.T) {
 	assert.NoError(t, err) // This should work fine for pods
 }
 
+// TestModelServerController_SharedPods tests a scenario where:
```

Comment:
> Not sure i understand what you say. When sync later modelserver indicates what

@hzxuzhonghu UT might be easier to understand.

Reply:
Still not understand